solutions/standard/main.tf

```diff
@@ -194,3 +191,70 @@ module "event_notifications" {
   skip_en_cos_auth_policy = var.skip_en_cos_auth_policy
   cos_endpoint            = local.cos_endpoint
 }
+
+# create a service authorization between Secrets Manager and the target service (Event Notification)
+resource "ibm_iam_authorization_policy" "secrets_manager_key_manager" {
+  count                       = var.skip_en_sm_auth_policy || var.existing_secrets_manager_instance_crn == null ? 0 : 1
+  depends_on                  = [module.event_notifications]
```

Comment:
why the explicit depends_on ? There is already an implicit dependency on line 202 where you pass the GUID of the event notifications instance